hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -539,7 +595,7 @@ timedTx tmpDir tracer node@RunningNode{networkId, nodeSocket} hydraScriptsTxId =
     send n1 $ input "Init" []
     headId <-
       waitForAllMatch 10 [n1] $
-        headIsInitializingWith (Set.fromList [alice])
+        headIsInitializingWith (Set.fromList [aliceTimed])
```

Comment:
I also just fixed those today. Instead of renaming them, we can just drop lines `528` and `529` and use the `alice` and `aliceSk` from the fixture.